components/network/memcached/Makefile

```diff
@@ -21,36 +21,51 @@
 
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2023, Friedrich Kink. All rights reserved.
 #
 
-BUILD_BITS=64
+BUILD_BITS=		64
+OPENSSL_VERSION=	3.1
+
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		memcached
-COMPONENT_VERSION=	1.6.2
+COMPONENT_VERSION=	1.6.22
+COMPONENT_SUMMARY=	The memcached memory object cache
+COMPONENT_DESCRIPTION=	Memcached is a high performance multithreaded event-based \
+			key/value cache store intended to be used in a distributed system.
+COMPONENT_FMRI=		service/memcached
+COMPONENT_CLASSIFICATION=Web Services/Application and Web Servers
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_PROJECT_URL=	http://memcached.org/
+COMPONENT_PROJECT_URL=	https://memcached.org/
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_HASH=	\
-    sha256:06720118c40689be0b85249b3dcb23c6e6d5e3ce53893aca9faced264145168b
-COMPONENT_ARCHIVE_URL=	http://www.memcached.org/files/$(COMPONENT_ARCHIVE)
+    sha256:34783a90a4ccf74c4107085fd92b688749d23b276cfdad9f04e4f725a05d1ca7
+COMPONENT_ARCHIVE_URL=	https://www.memcached.org/files/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=	BSD
+COMPONENT_LICENSE_FILE=	COPYING
 
 include $(WS_MAKE_RULES)/common.mk
 
 COMPONENT_PRE_CONFIGURE_ACTION = \
-	($(CLONEY) $(SOURCE_DIR) $(@D))
+	(cd $(SOURCE_DIR); aclocal; autoconf; $(CLONEY) $(SOURCE_DIR) $(@D))
 
 COMPONENT_TEST_TARGETS=	test
 
-CFLAGS += -D__ATTRIBUTE_DISABLED
+CFLAGS += -I/usr/include/libevent2 -D__ATTRIBUTE_DISABLED
 
-CONFIGURE_BINDIR.64 =	$(CONFIGURE_PREFIX)/lib
-CONFIGURE_OPTIONS.64 +=	--with-libevent=/usr/lib/${MACH64}
+CONFIGURE_OPTIONS +=	--with-libevent=/usr/lib/${MACH64}
 CONFIGURE_OPTIONS +=	--enable-dtrace
+CONFIGURE_OPTIONS +=	--disable-sasl
+#CONFIGURE_OPTIONS +=	--enable-asan
+#CONFIGURE_OPTIONS +=	--enable-werror
```

Comment:
Why these formerly enabled options have been disabled? Please write at least some explanation so we don't have to guess.